src/find-user.ts

```diff
@@ -24,3 +24,11 @@ export async function findUserById(id: number): Promise<User> {
   }
   return idUser;
 }
+export async function listUsers(toList: number) {
+  const listUsers = await AppDataSource.getRepository(User)
+    .createQueryBuilder('user')
+    .orderBy('user.name')
+    .take(toList)
+    .getMany();
+  return listUsers;
```

Comment:
```suggestion
  return AppDataSource.getRepository(User)
    .createQueryBuilder('user')
    .orderBy('user.name')
    .take(toList)
    .getMany();
```

Geralmente consts que vc já retorna abaixo podem ser substituidas pelo retorno direto. E por conta de o `getMany` já retornar uma promise, vc poderia remover o async, nesse caso.